src/java/picard/sam/markduplicates/util/ReadEndsForMarkDuplicates.java

```diff
@@ -34,10 +34,10 @@
     What do we need to store you ask?  Well, we need to store:
        - byte: orientation
        - short: libraryId, readGroup, tile, x, y, score
-       - int: read1ReferenceIndex, read1Coordinate, read2ReferenceIndex, read2Coordinate
+       - int: read1ReferenceIndex, read1Coordinate, read2ReferenceIndex, read2Coordinate, duplicateSetSize
        - long: read1IndexInFile, read2IndexInFile
      */
-    protected static final int SIZE_OF = (1 * 1) + (5 * 2) + (4 * 4) + (8 * 2) + 1
+    protected static final int SIZE_OF = (1 * 1) + (5 * 2) + (5 * 4) + (8 * 2) + 1
```

Comment:
The first series of entries are the number primitives of each type that will be stored in the object multiplied by the number of bytes needed for each type: 
http://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html

Here is info I was reading about the overhead of 8 bytes listed here: 
http://stackoverflow.com/questions/258120/what-is-the-memory-consumption-of-an-object-in-java

I'm not sure how the extra 13 bytes were calculated before.